### ISSUE TITLE
menu_item crashes app with SIGSEV

### ISSUE THREAD

**Issue by oxysoft** [pthom/imgui_bundle#254]
```
import imgui
from imgui_bundle import imgui_ctx, immapp

def gui():
	with imgui_ctx.begin_main_menu_bar():
		with imgui_ctx.begin_menu("File"):
			clicked,_ = imgui.menu_item("New")

if __name__ == '__main__':
	immapp.run(gui)
```

Switch up the menu_item with pass and the app launches fine.

**Comment by oxysoft** [comment]
*facepalm* It was in the imports, I had another set of imgui bindings installed and accidentally imported that with pycharm. (note the imports here)

**Comment by pthom** [comment]
Hello,

Thanks for updating me :-)

If you have a few minutes, I'd be interested to have more information about the context in which you are using imgui bundle. 
I'm always curious to know what users are doing with it ;-)

**Comment by oxysoft** [comment]
Of course! I make visionary AI animation, you can see my work on instagram https://www.instagram.com/ryunuck_/ I made my own software for this which is a pseudo-gui in pygame, kind of like MPV where the whole user interface is handmade

![2023-05-21](https://github.com/user-attachments/assets/21c27fc9-9ad7-4630-a7ec-0eacbcaa54c8)

This lets me play back the image sequence as though it were a video, and is generally just very optimized for the workflow. I then eventually bolted a graphing calculator onto it with pyqtgraph so I could inspect every parameter for each frame - the dynamics involved are complex and many render parameters co-interact, so it's crucial to know in advance what that will look like considering it takes about 1 hour to render one minute of animation.

Problem is, I now want to do a lot more with it, make the graph into a curve editor so I can program camera movement and whatnot. It's a total clusterfuck. I have no idea how to use PyQt productively, there are so many layers to this and so many quirks and strange non-intuitive behaviors that I more or less gave up.

Dear ImGUI on the other hand, I know this like the back of my hand. Never used the plotting stuff so I'm still not sure how flexible it will be, but I've made an entire MIDI editor with a piano roll using only custom DrawList and maths, so I know it's gotta be possible to bend it pretty good.
